packages/wrangler/src/__tests__/package-manager.test.ts

```diff
@@ -175,7 +175,7 @@ const testCases: TestCase[] = [
 		npmLockFile: false,
 		yarnLockFile: false,
 		pnpmLockFile: false,
-		expectedPackageManager: "npm",
+		expectedPackageManager: "pnpm",
```

Comment:
How come this test changes even though there is no change to package-manager.ts?

Reply:
Oh it's because the logic has `// check the user agent` before `// check what's installed`.
So it picks up on the fact that the tests are being run under `pnpm`.
This is actually a bug in how the tests are being setup, since we should also be controlling the user agent being passed to the function, I think.